docker-compose.yml

```diff
@@ -36,6 +36,7 @@ x-kuzzle-config: &kuzzle-config
     - KUZZLE_SECRETS_FILE=${KUZZLE_SECRETS_FILE:-/var/app/features/fixtures/secrets.enc.json}
     # Variables used by the development scripts
     - NODE_12_VERSION=12.13.0
```

Comment:
Well, I know it's not yours but this version of node is not even present in our matrix `node-version: ['14.15.4', '12.16.3', '12.20.0']`